src/Command/InfectionCommand.php

```diff
@@ -174,6 +176,10 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        if (!$this->isDebuggerOrAdditionalParametersExist()) {
+            return 1;
```

Comment:
Should not we output the error message here directly, but not as a side effect of calling `isDebuggerOrAdditionalParametersExist`?